tests/unit/vector_tile.test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <vector_tile.hpp>
+
+#include <iostream>
+#include <cassert>
+
+using namespace vector_tile;
+
+static void test_config() {
+    assert(Encoding::Tile::LAYERS == 3);
```

Comment:
Let's move to using `catch.hpp` for unit tests for this project (by vendoring the standalone header from https://github.com/philsquared/Catch/blob/master/single_include/catch.hpp). The problem with `assert` is that it will be removed by the compiler if the project is not build with `-DNDEBUG`.